free-1.md

```diff
@@ -57,12 +173,30 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I suspect that a lot of time would be dedicated into semantics, specifically
+into calculating the spacing between branches.
```

Comment:
I agree - I think that there is a lot of systems work necessary to get things working.